styles/_search_results.scss

```diff
@@ -80,6 +92,34 @@
       }
     }
   }
+
+  .meta-group {
+    display: flex;
+    align-items: center;
+    clear: right;
+    color: $field_text_color;
+
+    li:first-child {
+      margin-right: auto;
+    }
+
+    .meta-data {
+      color: inherit;
+
+      &:not(:last-child) {
+        margin-right: 20px;
```

Comment:
There's several `margin-right` applied in this PR. Usually needs the opposite for `rtl`.
Are we adding support for `rtl` in a following PR?

Reply:
good catch. I actually haven't thought about `rtl` yet. do we have an example to handle `rtl` in the css?

Reply:
Here's an example of us swapping the `right` specific styles with `left` for `rtl` support:
https://github.com/zendesk/copenhagen_theme/blob/master/style.css#L1460-L1468